src/Card/AddCard.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+import styles from './AddCard.module.css';
+const { v4: uuidv4 } = require('uuid');
+
+const AddCard = props => {
+    const [title, setTitle] = useState('');
+    const [description, setDescription] = useState('');
```

Comment:
Why use 2 states? Better merge one, then use dynamic properties tobind it to onChange event handler and update state properly.